source/js/interview.js

```diff
@@ -1,277 +1,258 @@
-            setTimeout(() => {
-                elementScrollToTop("modal-box");    
-            }, 900);
-        }, 680);
-    }
 
-    // Function that's responsible for calling the cardTogglePhone function with the right parameters
-    function cardPhone(e) {
-        if (!isCardOpen)
-            cardTogglePhone(e);
-        else
-            cardTogglePhone(null);
-    }
+  /* Local Functions Section */
 
-    // Function that's responsible for opening up the modal box and freezing the background
-    function cardDesktop(e) {
-        // Check if card ain't already opened
-        if (!isCardOpen)
-            cardToggleDesktop(e);
-        else
-            cardToggleDesktop(null);
-        // Freezing background
-        $("body").toggleClass("no-overflow");
-        menu.toggleClass("no-display");
-        modal_background.toggleClass("no-display");
-        closeBtn.toggleClass("no-display");
-        modal_box.toggleClass("no-display");
-        // Scrolling to the top of the modal box
-        elementScrollToTop("modal-box");
-        isCardOpen = !isCardOpen;
-        // Last check whether our card opened correctly 
-        if (isCardOpen)
-            currentCard = e.id;
-        else
-            currentCard = null;
-        
-        // Changing the active tab in the modal navigation to the current one
-        if(currentCard && currentCard != null)
-            setActiveNavTab(currentCard);
-    }
+  /** Changes the content inside our modal box when user clicks on a tab in our interview menu
+  *
+  // eslint-disable-next-line max-len
+  * @param {HTMLElement} sender an Html element that is considered the sender to extract necessary things
+  */
+  function changeCard(sender) {
+    modifyUrl(hashCode(sender.id.replace('_menu', '')));
+    modal_box.animate({ height: '0px' }, 200).delay(680).animate({ height: '100vh' }, 200);
+    setTimeout(() => {
+      changeCurrentCardContent(sender);
+      setTimeout(() => {
+        elementScrollToTop('modal-box');
+      }, 900);
+    }, 680);
+  }
 
-    // Function that's responsible for scrolling to the top of the given element
-    function elementScrollToTop(id) {
-        try{
-            document.getElementById(id).scrollTo({
-                top: 0,
-                left: 0,
-                behavior: 'smooth'
-              });
-        }
-        catch(err){
-            document.getElementById(id).scrollTop = 0;
-        }
-    }
+  // Function that's responsible for calling the cardTogglePhone function with the right parameters
+  function cardPhone(e) {
+    if (!isCardOpen) { cardTogglePhone(e); } else { cardTogglePhone(null); }
+  }
 
-    // Function that's responsible for changing the active tab in our interview menu
-    function setActiveNavTab(ID) {
-        var children = menu.children().children();
+  // Function that's responsible for opening up the modal box and freezing the background
+  function cardDesktop(e) {
```

Comment:
this is confusing naming - it's a function, it does stuff, it doesn't make sense to name it by a noun phrase! what does it actually do?